README.md

```diff
@@ -72,6 +72,7 @@ A curated list of awesome things and projects built by Algerian developpers.
 ## Tutorials and Learning Resources 
 - [Computer vision tutorial (OpenCV & python)](https://github.com/amineHorseman/opencv-eldjmaa-live-coding) - Live coding session for learning computer vision, from capturing images to detecting faces (link to videos + code)
 - [Frontend-Learning-Journey](https://github.com/Amine-Smahi/Frontend-Learning-Journey) - Tutorials, definitions, frameworks and sample projects if you want to learn frontend web developement 
+- [Python Code Tutorials](https://github.com/x4nth055/pythoncode-tutorials) - Python programming tutorials and recipes for beginner and intermediate Python programmers.
```

Comment:
Hey, thanks for the addition. Will you consider adding a license file to your repository?

Reply:
Done.
Thanks!